index.bs

```diff
@@ -634,6 +634,15 @@ Each {{XRSession}} has a <dfn>minimum inline field of view</dfn> and a <dfn>maxi
 
 Each {{XRSession}} has a <dfn>minimum near clip plane</dfn> and a <dfn>maximum far clip plane</dfn>, defined in meters. The values MUST be determined by the user agent and MUST be non-negative. The [=minimum near clip plane=] SHOULD be less than <code>0.1</code>. The [=maximum far clip plane=] SHOULD be greater than <code>1000.0</code> (and MAY be infinite).
 
+<div class="algorithm" data-algorithm="update-layers-state">
+
+When the user agent will <dfn>update the pending layers state</dfn> with {{XRSession}} |session| and {{XRRenderState}} |state|, it must run the following steps:
```

Comment:
XRRenderStateInit newState

Reply:
Done. Sorry that I missed your comments.